modules/etcd-overview.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+
+
+:_content-type: CONCEPT
+[id="etcd-overview_{context}"]
+= Overview of etcd
+
+etcd is a consistent, distributed key-value store that holds small amounts of data that can fit entirely in memory. Although etcd is a core component of many projects, it is the primary data store for Kubernetes, which is the standard system for container orchestration.
+
+[id="etcd-benefits_{context}"]
+== Benefits of using etcd
+
+By using etcd, you can enjoy several benefits:
+
+* Safely and automatically update containers
+* Coordinate work that is scheduled to hosts
+* Set up overlay networking for containers
```

Comment:
out of curiosity, where did you find the above three bullet? The below ones are spot on.

Reply:
Those three bullets are based on information I found on this page: https://www.redhat.com/en/topics/containers/what-is-etcd. See the first paragraph.

Reply:
Gotcha. I think this doc needs to be clear what etcd itself does and what it helps others to facilitate. The above three bullets are being done by operators and controllers in OpenShift, etcd merely stores. Below three bullets are features of etcd itself.

Reply:
Good to know -- thanks! With that in mind, I'll remove the first 3 bullets.